raw_radar_reader/README.md

```diff
@@ -0,0 +1,60 @@
+# Raw Radar Interface
+
+This directory contains tools for interacting with raw radar data. Zendar
+makes available raw ADC samples from our FMCW radars. The example tools
+in this directory allow researchers to easily use this data format.
+
+o## Dependencies
+```
+libprotobuf
+protobuf-compiler
+```
+
+Python packages:
+```
+pip3 install -r requirements.txt
+```
+
+## Setup
+
+To use these tools, perform the following steps:
+```
+source environment.sh
```

Comment:
Once https://github.com/ZendarInc/ZendarSDK/pull/27 is merged, we will need to provide instructions on generating the Python ```_pb2``` files from the protobuf files. We need this for ```zendar-zenapi_amd66.deb``` since this is the package containing all ```.proto``` files.

Reply:
> > @osamasal yeah would love to move over to the deb thing asap. When is that expected to land?
> 
> Should be very soon. I have the [PR](https://github.com/ZendarInc/ZendarSDK/pull/27) out for review and I'm just waiting for sign-off before merging. In this PR, I updated the CPP examples, but I'll update and fix the existing Python tools under `ZendarSDK/python/viewer` shortly after. My ETA for merging this PR is early week of 03/07/2021.

@antoniopug https://github.com/ZendarInc/ZendarSDK/pull/27 has been merged, so the protos are now part of the ```zendar-zenapi.deb``` package.